Pod/Classes/SEGAdobeIntegration.h

```diff
@@ -28,6 +28,27 @@
 @interface SEGRealADBMediaObjectFactory : NSObject <SEGADBMediaObjectFactory>
 @end
 
+@protocol SEGPlaybackDelegate <NSObject>
+@end
+
+
+@interface SEGPlaybackDelegate : NSObject <ADBMediaHeartbeatDelegate>
+- (NSTimeInterval)getCurrentPlaybackTime;
+- (void)updatePlayheadPosition:(long)playheadPosition;
+- (void)resumePlayheadAfterSeeking;
+- (void)unPausePlayhead;
+- (void)pausePlayhead;
+- (void)incrementPlayheadPosition;
+@end
+
+@protocol SEGPlaybackDelegateFactory <NSObject>
+- (SEGPlaybackDelegate *_Nullable)createPlaybackDelegate;
+@end
+
+
+@interface SEGRealPlaybackDelegateFactory : NSObject <SEGPlaybackDelegateFactory>
```

Comment:
I think this should be doing something - otherwise the real one will always return nil.